Sources/BrowserServicesKit/ContentBlocking/ContentBlockerRulesManager.swift

```diff
@@ -388,6 +386,15 @@ public class ContentBlockerRulesManager: CompiledRuleListsSource {
                                                                                                 unprotectedSitesHash: nil))
             }
 
+            // todo - this hard code is a placeholder till I figure out how to check for this task vs. Ad attr task
```

Comment:
I think I saw an enum for this somewhere, but cannot recall. What is the best way to check for the correct task?